app/helpers/script_helper.rb

```diff
@@ -2,8 +2,8 @@
 
 # rubocop:disable Rails/HelperInstanceVariable
 module ScriptHelper
-  def javascript_packs_tag_once(*names, **attributes)
-    @scripts = @scripts.to_h.merge(names.index_with(attributes))
+  def javascript_packs_tag_once(*names, **attributes_and_url_params)
```

Comment:
could we make `url_params` an optional keyword so we don't have to munge the `attributes_and_urls_params` together?
```suggestion
  def javascript_packs_tag_once(*names, url_params: nil, **attributes)
```

Reply:
Had to rearrange things a bit with how this carries over between the enqueue and the render, but should be better now in c05c3e0b18. I like 👍